module/README.md

```diff
@@ -0,0 +1,102 @@
+# Module atracker
```

Comment:
Create a directory under modules and copy the respective files like main.tf, variables.tf, versions.tf etc which currently under modules directory

Reply:
https://github.com/terraform-ibm-modules/terraform-ibm-atracker/pull/2#:~:text=update%20module%20directory-,cf33db0,-Merge%20state